lnst/Recipes/ENRT/GreTunnelOverBondRecipe.py

```diff
@@ -161,20 +165,19 @@ def create_tunnel(self, configuration):
 
         # A
         m1.gre_tunnel.up()
-        m1.gre_tunnel.ip_add(a_ip4)
-        m1.gre_tunnel.ip_add(a_ip6)
+        config.configure_and_register_ip(m1.gre_tunnel, a_ip4)
+        config.configure_and_register_ip(m1.gre_tunnel, a_ip6)
         m1.run("ip -4 route add {} dev {}".format(b_net4, m1.gre_tunnel.name))
         m1.run("ip -6 route add {} dev {}".format(b_net6, m1.gre_tunnel.name))
 
         # B
         m2.gre_tunnel.up()
-        m2.gre_tunnel.ip_add(b_ip4)
-        m2.gre_tunnel.ip_add(b_ip6)
+        config.configure_and_register_ip(m2.gre_tunnel, b_ip4)
+        config.configure_and_register_ip(m2.gre_tunnel, b_ip6)
         m2.run("ip -4 route add {} dev {}".format(a_net4, m2.gre_tunnel.name))
         m2.run("ip -6 route add {} dev {}".format(a_net6, m2.gre_tunnel.name))
 
-        configuration.tunnel_devices.extend([m1.gre_tunnel, m2.gre_tunnel])
-        self.wait_tentative_ips(configuration.tunnel_devices)
```

Comment:
You removed this from the code, so we will now won't wait for tentative flag.

Reply:
Accidentally removed, I returned it back there.